module.nix

```diff
@@ -123,6 +123,29 @@ in
       };
     };
 
+    # RAC oupost
+    authentik-rac = {
+      enable = mkEnableOption "authentik RAC outpost";
+
+      environmentFile = mkOption {
+        type = types.nullOr types.path;
```

Comment:
`types.path` isn't a good idea for secret files. I fixed the other occurrences in https://github.com/nix-community/authentik-nix/pull/51, so it's probably the easiest to rebase onto this PR and use `pathToSecret`.